documentation/metadata-editing.md

```diff
@@ -0,0 +1,87 @@
+# Editing metadata of resources
+
+## What is a metadata resource
```

Comment:
There should be a link to https://solidproject.org/TR/2021/protocol-20211217#auxiliary-resources-description-resource somewhere.